packages/react/src/components/StructuredList/StructuredList.tsx

```diff
@@ -315,7 +315,8 @@ export function StructuredListRow(props: StructuredListRowProps) {
       onKeyDown={onKeyDown}>
       <GridRowContext.Provider value={value}>
         {selection && (
-          <StructuredListCell>
+          <StructuredListCell
+            className={`${prefix}--structured-list-icon-column-first`}>
```

Comment:
We don't want to require the user to have to add a new classname to get the correct styles. 
```suggestion
          <StructuredListCell>
```

Reply:
Changes were required on both feature flag version of Structured list and the regular version.
Updated the PR to use pseduoselectors instead of new classes. 
Thanks for the review @alisonjoseph  💯